### PR TITLE
update 0.1.0 version and doc

### PR DESCRIPTION
**Pull request by yiyione**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,61 +13,69 @@ It could be used to support existing or future front-ends (e.g. Web Portal, VS C
...
+
     ```bash
     myenv/Scripts/activate
     pai -h
     ```
+
 - use a absolute path to `pai`
+
     ```bash
     myenv/Scripts/pai -h
```
Comment (mydmdm) [inline README.md:56]: `myenv/bin/pai -h`

File: README.md [hunk 2]
```diff
@@ -12,61 +13,69 @@ It could be used to support existing or future front-ends (e.g. Web Portal, VS C
...
-npm i -g microsoft/openpaisdk
+npm i -g @microsoft/openpai-js-sdk
 ```
 
 This installation commands will generate a virtual environment with latest `node` in the directory `./myenv`, and install the CLI in it. Then user could use `pai` command by any of below methods
 
 - activate the virtual environment first
+
     ```bash
     myenv/Scripts/activate
```
Comment (mydmdm) [inline README.md:49]: `source myenv/bin/activate`

File: README.md [hunk 3]
```diff
@@ -12,61 +13,69 @@ It could be used to support existing or future front-ends (e.g. Web Portal, VS C
...
     ```
+
 - use a absolute path to `pai`
+
     ```bash
     myenv/Scripts/pai -h
     ```
-- add `myenv/Scripts` to environment variable `path`
 
+- add `myenv/Scripts` to environment variable `path`
```
Comment (mydmdm) [inline README.md:59]: `myenv/bin`

File: README.md [hunk 4]
```diff
@@ -18,18 +19,20 @@ npm install --save microsoft/openpaisdk
 Initialize the `openPAIClient`
 
 ```ts
-const cluster: IPAICluster = {
+import { PAIV2 } from '@microsoft/openpai-js-sdk';
+
+const cluster: PAIV2.IPAICluster = {
```
[the remark this answers is not in the document]
Comment (yiyione) [reply README.md:24]: Done.

File: README.md (hunk 1 above)
Comment (yiyione) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (yiyione) [reply README.md]: Done.

File: README.md (hunk 3 above)
Comment (yiyione) [reply README.md]: Done.

**Pull request merged**